test/test.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import { spy } from 'sinon';
+import { createMockStore } from 'redux-test-utils';
+import ValidatorsList from '../src/components/ValidatorsList.react';
+
+spy(ValidatorsList.prototype, 'componentDidMount');
```

Comment:
I'm not familiar with enzyme, but looks interesting!

Reply:
Are you familiar with jest or any other? We chose this one because there were lots of guides to use it with mocha, but it would be nice to hear other experiences before we dig too deep.

Reply:
Not really. I've only worked with tests for reducers, not components, using vanilla mocha (kuak).